src/pages/productList/ProductList.jsx

```diff
@@ -0,0 +1,142 @@
+import prodList from "./productList.module.css";
+import filter from "./filter.module.css";
+import { categories } from "../home/Home";
+import { v4 as uuid } from "uuid";
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+export function ProductList() {
+  const [productList, setProductList] = useState([]);
+
+  useEffect(() => {
+    (async function () {
+      try {
+        let products = await axios.get("/api/products");
+        console.log(products);
+        setProductList(products.data.products);
+      } catch (err) {
+        console.log(err);
+      }
```

Comment:
Remove console log, show the error in UI.

Reply:
done